ui/components/ui/icon/paper-airplane-icon.js

```diff
@@ -22,7 +22,7 @@ export default function PaperAirplane({ size, className, color }) {
 }
 
 PaperAirplane.defaultProps = {
-  color: '#FFFFFF',
+  color: 'var(--color-background-default)', // @TODO: Keep white or change to icon muted ?
```

Comment:
I just did a search for this component to get an idea of the context and it's not used anywhere. So we can probably remove it. You could do this in this PR or for a quick fix we can just add icon-default here

```suggestion
  color: 'var(--color-icon-default)',
```

Reply:
Will remove it :)